src/ocean/time/Time.d

```diff
@@ -122,29 +139,23 @@ struct TimeSpan
          */
         long ticks()
         {
-                return ticks_;
+            return this.impl.total!"hnsecs";
```

Comment:
What's the basis for assuming that `ticks == hnsecs` is valid?  It's not at a glance an obvious equivalence, so it would be good to have a comment (and ideally, reference) explaining the validity.

Reply:
An hecto-nanoseconds is 100ns (https://dlang.org/phobos/core_time.html).
And we have this constant: https://github.com/sociomantic-tsunami/ocean/pull/809/files#diff-77aa1635ec08d59b7a587a49057b8b0fR79

Reply:
Gotcha.  Would you be happy to add a small cross-ref to those definitions as a code comment before the `return` statement?  It would make the relationship more obvious at first glance.

(You could probably just make it a cross-ref to the `NanosecondsPerTick` value above, as you only really have to show consistency with ocean.)